### PR TITLE
MIP61: abstain modifier

### PR DESCRIPTION
**Pull request by ultraschuppi**
https://forum.makerdao.com/t/mip4c2-spxx-amend-mip61-to-tighten-up-recognized-delegate-participation-metrics/18696

### CODE REVIEW

File: MIP61/MIP61.md [hunk 1]
```diff
@@ -77,6 +78,10 @@ Vote outcomes are largely determined by how well voters are informed. As Recogni
 
 The intention behind the Performance Modifier is that these parameters be considered over the duration of time that a Recognized Delegate is active. The advantage of such an approach is that it allows Recognized Delegates that have been performing at high levels flexibility in terms of unexpected ability to vote, for example, in the event of illness, or planned inactivity, such as a vacation period. If a Recognized Delegate's parameters were calculated on a rolling-basis, for example over three months, taking a week off would have an outsized effect on compensation, particularly if it was a week with a large number of Governance Polls. If it is difficult or uneconomical for Recognized Delegates to take time off when required, there is potential this could lead to burnout and this is consequently undesirable.
 
+### Abstain Modifier Motivation
+
+The intention behind the Abstain Modifier is to avoid the problem of getting compensated even if a delegate is constantly just abstaining on polls. To a certain extend it can be expected that delegates take a position on the governance questions.
```
Comment (ghost) [inline MIP61/MIP61.md]: Suggested slight rewording, grammar, and clarification

```suggestion
The intention behind the Abstain Modifier is to avoid the problem of getting compensated even if a delegate is constantly just abstaining on polls. To a certain extent, Recognized Delegates should be expected to take a position on governance questions and Executive Votes. However, it is recognized that in certain situations, Recognized Delegates may have good reasons to abstain from a specific vote.
```

File: MIP61/MIP61.md [hunk 2]
```diff
@@ -129,6 +134,10 @@ This is demonstrated in the below graph:
 
 These requirements and metrics may be adjusted in the future depending on the number of Recognized Delegates, the resources available to the Governance Facilitators, and feedback from the Recognized Delegates themselves.
 
+#### Abstain Modifier
+
+At the beginning of every month Governance Facilitators check how many of the polls in the previous month have been voted with a non-abstaining choice. If the ratio of non-abstaining-choices to total polls is >= 60% the delegate met the Abstaind Modifier criteria and is eligable for compensation. If the ratio is below 60%, the delegate is not eligable for any compensation for this month.
```
Comment (ghost) [inline MIP61/MIP61.md]:
```suggestion
At the beginning of every month, the Governance Facilitators will check how many polls and Executive Votes in the previous month have been voted with a non-abstaining choice. If the ratio of non-abstaining choices to total polls is >= 60%, the Recognized Delegate will have met the Abstain Modifier criteria and be eligible for compensation for that month. If the ratio is below 60%, the delegate will not be eligible for compensation for that month.
```

**Review by ghost — COMMENTED** [review]
Suggested some changes

File: MIP61/MIP61.md (hunk 1 above)
Comment (ultraschuppi) [reply MIP61/MIP61.md]: i mean, this does not really apply to executives right? i think it is perfectly okay to support no executive if proper reasoning is provided.

Comment (ghost) [reply MIP61/MIP61.md]: Agree, but IMO RDs should still be taking a position, which could be abstain.

Comment (ultraschuppi) [reply MIP61/MIP61.md]: abstain on a executive? how is that different from not supporting it?

image

this is why i am asking, i would say this is all about polls

Comment (ghost) [reply MIP61/MIP61.md]: So *technically* there's a difference between abstaining and not supporting. At the smart contract level it's the same result (unless someone moves MKR to make it more difficult for an exec to get the hat). But at the social level there is a distinct difference, no? Ultimately it's not a particularly big deal so feel free to word it however you like 😄

Comment (ultraschuppi) [reply MIP61/MIP61.md]: i guess i gonna reword it to just catch polls and not execs. 

execs still feel pretty binary to me
- you either support it by adding your MKR to it withing 72h
- you do not support it by leaving your MKR on the previous hat (which is likely the most powerful move to boycott the new spell) - and communicate your reasons within 72h

**Comment by blimpa** [comment]
Merged as per [Ratification Poll for Amend MIP61 to tighten up Recognized Delegate Participation Metrics (MIP4c2-SP29) - January 9, 2023](https://vote.makerdao.com/polling/Qmd2W3Q4).
